pkg/prometheus/statefulset.go

```diff
@@ -98,6 +101,16 @@ func makeStatefulSet(p monitoringv1.Prometheus, old *appsv1.StatefulSet, config
 		p.Spec.Retention = defaultRetention
 	}
 
+	if p.Spec.LivenessFailureThreshold == nil || p.Spec.LivenessFailureThreshold < 1 {
```

Comment:
Shouldn't that be `< minLivenessFailureThreshold`? Or should the var be named `defaultLivenessFailureThreshold`?